src/interfaces/signSettings.ts

```diff
@@ -9,7 +9,20 @@ export interface SignSettings {
   runtimeOptions?: Partial<Board.RpiLedMatrix.RuntimeOptions>;
   mqttOptions: SignMqttOptions;
   defaultState: DefaultState;
-  canvasSections: CanvasSectionSettings[];
+  canvasSections: CustomCanvasSectionSettings[];
+}
+
+export interface CustomCanvasSectionSettings
+  extends Omit<CanvasSectionSettings, "representation"> {
+  representation: CustomPaintingInstruction[];
+}
+
+export interface CustomPaintingInstruction
+  extends Omit<Board.PaintingInstruction, "text"> {
+  text?: {
+    value: string;
+    replaceWithDateTime?: boolean;
```

Comment:
Figured we would need some more options for text, hence all this TS Omits.
There is a function `canvasSectionsWithReplacedValues` that will translate the data fom these custom types back into the native `PaintingInstruction` (which would be required for all text calls).

Maybe we can avoid this by defining a subsitution string pattern such as `$foobar` or %%dd-mm-yyyy%% for date/time subsitutions? See `canvasSectionsWithReplacedValues`.